src/components/ErrorList.jsx

```diff
@@ -10,10 +10,10 @@ function ErrorList({errors, onErrorClick, language}) {
     return false;
   }
 
-  const errorItems = map(errors.items, error => (
+  const errorItems = map(errors.items, (error, index) => (
     <ErrorItem
       {...error}
-      key={[error.reason, error.row]}
+      key={[error.reason, error.row, index]}
```

Comment:
If we’re going to use the `index` in the `key`, I don’t think there’s any reason to still use the `reason` and `row`, since the `index` alone provides a unique (if not stable) identifier for each error.

However [using an array index in a key is generally discouraged](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318). The underlying problem here is that the `reason` is not sufficiently unique—in particular, there are two different `unclosed-tag` errors, for two different tags, but on the same line.

If instead of `error.reason` we used `error.raw` (a string containing the full error message in Markdown), I think we’d be in good shape while avoiding the problems with using an index?